packages/vhd-lib/index.js

```diff
@@ -14,3 +15,6 @@ exports.VhdDirectory = require('./Vhd/VhdDirectory').VhdDirectory
 exports.VhdFile = require('./Vhd/VhdFile').VhdFile
 exports.VhdSynthetic = require('./Vhd/VhdSynthetic').VhdSynthetic
 exports.Constants = require('./_constants')
+const {isVhdAlias, resolveAlias} = require('./_resolveAlias')
+exports.isVhdAlias = isVhdAlias
+exports.resolveAlias = resolveAlias
```

Comment:
I believe we should eventually remove this index in favor of specific submodule exports/imports.

Example for this commit:

```js
import { isVhdAlias, resolveAlias } from 'vhd-lib//aliases.js'
```